tests/unit_tests/test_runner.py

```
@@ -14,6 +14,36 @@
 from casanovo.denovo.model_runner import ModelRunner
 
 
+def test_loading_timstof_folders(tmp_path, monkeypatch):
+    # Mocking constructor of ModelRunner
+    def minimal_init(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `minimal_init`

Add return type annotation: `None`

(ANN202)